Dockerfile

```diff
@@ -19,9 +19,20 @@ RUN apk add --no-cache libcurl procps sqlite-libs && \
 WORKDIR /wpscan
 RUN rake install --trace
 
+FROM ruby:2.5-alpine
+LABEL maintainer="WPScan Team <team@wpscan.org>, Mostafa Hussein <mostafa.hussein91@gmail.com>"
```

Comment:
Please leave the maintainer as WPScan. There is a difference between contributor to the image and maintainer of the image